xmake/rules/utils/ispc/ispc.lua

```diff
@@ -0,0 +1,77 @@
+rule("utils.ispc")
+    set_extensions(".ispc")
+    add_deps("utils.inherit.links")
+
+    on_load(function (target)
+        local header_outputdir = path.join(target:autogendir(), "ispc_headers")
+        local obj_outputdir = path.join(target:autogendir(), "ispc_objs")
+        os.mkdir(target:autogendir())
```

Comment:
we should use `batchcmds:mkdir` in before_buildcmd_file, only obj_outputdir need be create here